lib/src/Pos/Web/Server.hs

```diff
@@ -6,6 +6,7 @@
 
 module Pos.Web.Server
        ( serveImpl
+       , withRoute53HealthCheckApplication
        , route53HealthCheckApplication
```

Comment:
`route53HealthCheckApplication` is no longer used outside this module, so I want it to be removed from the export list.

Reply:
Sure